src/constants/schema.js

```diff
@@ -23,8 +23,8 @@ export const REDIRECT_ITEM =
 
 export const MESSAGE_IN_APP =
   "https://ns.adobe.com/personalization/message/in-app";
-export const MESSAGE_FEED_ITEM =
-  "https://ns.adobe.com/personalization/message/feed-item";
+export const MESSAGE_CONTENT_CARD =
```

Comment:
`Message Feed Item` was renamed to `Content Card`?

Reply:
Yep, "Feed item" isn't a thing anymore.